### PR TITLE
Modular Inverse using Euclidean Division

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: content/4_Gold/Modular.mdx [hunk 1]
```diff
@@ -287,6 +287,35 @@ assert 2 * x % MOD == 1
 
 </LanguageSection>
 
+### With Euclidean Division
+
+We can also find the modular inverse from Euclidean Division. Given the prime modulus $m > a$ we have:
+$m = k \cdot a + r$, where k = $floor(\frac{m}{a})$	and $r = m % a$. Then: $0 = k \cdot a + r \mod m \iff r = -k \cdot a \mod m
```
Comment (envyaims) [inline content/4_Gold/Modular.mdx:293]:
```suggestion
$m = k \cdot a + r$, where k = $floor(\frac{m}{a})$	and $r = m \mod a$. Then: $0 = k \cdot a + r \mod m \iff r = -k \cdot a \mod m
```

File: content/4_Gold/Modular.mdx [hunk 2]
```diff
@@ -287,6 +287,35 @@ assert 2 * x % MOD == 1
...
+```cpp
+int inv(int x) {
+	if (x <= 1) { return x; }
+	return MOD - MOD / x * inv(MOD % x) % MOD;
+}
+```
+</CPPSection>
+</LanguageSection>
+
+The advantage of this approach is that we can precompute the inverse modular of numbers in range $[1, MOD)$ in $\mathcal{O}(MOD)$.
```
Comment (envyaims) [inline content/4_Gold/Modular.mdx:308]: theres a way easier way to compute the inverse?

**Review by envyaims — APPROVED** [review]
skibidi toilet

File: content/4_Gold/Modular.mdx (hunk 1 above)
Comment (brebenelmihnea) [reply content/4_Gold/Modular.mdx]: Ok done

**Review by ryanchou-dev — APPROVED** [review]

**Pull request merged**